test/Conversion/LinalgToTpp/linalg-to-tpp.mlir

```diff
@@ -64,12 +64,11 @@ func.func @add_mapping_scalar(%arg0: memref<f32>, %arg1: memref<f32>) {
 }
 
 // -----
-
 // We don't support broadcast for tpp.add. All operands must have the same type.
 #map = affine_map<(d0, d1) -> (d0, d1)>
 #map1 = affine_map<(d0, d1) -> (0, d1)>
 func.func @add_mapping_brcst(%arg0: memref<3x3xf32>, %arg1: memref<1x3xf32>) {
-  // CHECK-NOT: tpp.add
+// expected-error @below {{'linalg.generic' op result type not broadcast compatible with broadcasted operands's shapes}}
```

Comment:
we should not emit diagnostic here.

Reply:
Good point. It should just be lowered to non-TPP code.

Reply:
I thought the diagnostic came from verifyBroadcastableShapeImpl? I didn't introduce it obviously, I just use the check verifyBroadcastableShapeImpl in isTppAdd util. Is there a way to supress the diagnostic?

Reply:
yes, see: checkBroadcastableShape in https://github.com/plaidml/tpp-mlir/pull/424. If it is urgent I can try to split the PR.

Reply:
Its not urgent, we can wait till the other PR lands, no problem.

Reply:
Can we bypass the other PR to merge this first? IIUC, this is causing a performance regression on the benchmarks as it's not detecting the bias add and lowers to loops.

Reply:
We converged on the matcher design. I am planning to work on the PR now and hopefully merge it soon. If we want to land this first, we can take the changes for `checkBroadcastableShape`? Preference?